src/veritysetup/veritysetup.c

```diff
@@ -81,6 +82,35 @@ static int run(int argc, char *argv[]) {
                         return 0;
                 }
 
+                if (argc > 7 && strcmp(argv[7], "-")) {
```

Comment:
nit: !streq is generally preferred, as it's more readable

Reply:
Good to know. Thanks.